### PR TITLE
Feat(exporter-tokens): Document Themes naming rules #DS-1497

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

**Review by pavelklibani — APPROVED** [review]

File: exporters/tokens/README.md [hunk 1]
```diff
@@ -48,5 +48,16 @@ Once activated, the original data will be generated alongside other outputs and
...
 - \_original-groups.json
 
+## Naming Rules for Themes
+
+For the proper functionality of themes in the Spirit Tokens Exporter in Supernova, all theme names must adhere to specific conventions.
+Themes names are imported to Supernova from Figma, and designers are responsible for following these naming rules to ensure the correct
+export of design tokens.
+
+- Spaces are not allowed in theme names, as they disrupt the structure of the generated CSS (SCSS), which relies on precise syntax.
+- We recommend using a naming format such as `example-theme-light-default` to ensure consistency and compatibility.
```
Comment (crishpeen) [inline exporters/tokens/README.md:58]: i would leave out the example word
```suggestion
- We recommend using a naming format such as `theme-light-default` to ensure consistency and compatibility.
```

Comment (crishpeen) [reply exporters/tokens/README.md:58]: I know it only shows the format, but I think it is better to show even the content as intended.

**Review by crishpeen — APPROVED** [review]

**Comment by crishpeen** [comment]
Shouldn't the commit prefix be `Docs`?

**Comment by curdaj** [comment]
> Shouldn't the commit prefix be `Docs`?

Not sure, but I change commit massage to Docs

File: exporters/tokens/README.md [hunk 2]
```diff
@@ -48,5 +48,16 @@ Once activated, the original data will be generated alongside other outputs and
...
 - \_original-groups.json
 
+## Naming Rules for Themes
+
+For the proper functionality of themes in the Spirit Tokens Exporter in Supernova, all theme names must adhere to specific conventions.
+Themes names are imported to Supernova from Figma, and designers are responsible for following these naming rules to ensure the correct
+export of design tokens.
+
+- Spaces are not allowed in theme names, as they disrupt the structure of the generated CSS (SCSS), which relies on precise syntax.
+- We recommend using a naming format such as `theme-light-default` to ensure consistency and compatibility.
```
Comment (adamkudrna) [inline exporters/tokens/README.md:58]: Maybe mention this is called "kebab-case" syntax?

**Review by adamkudrna — APPROVED** [review]
👍🏻

**Pull request merged**